src/test/java/duke/TaskTest.java

```diff
@@ -0,0 +1,15 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class TaskTest {
+
+    @Test
+    public void getTask_getStringDescription_success() {
```

Comment:
Consider following the convention for writing testing methods: featureUnderTest_testScenario_expectedBehavior(). For this case, maybe can consider changing it into getTask_validTaskInput_success()